open_instruct/grpo_fast.py

```diff
@@ -328,6 +329,8 @@ class Args:
     on the first node and 4 learner processes on the second node; each process will have 1 GPU)"""
     vllm_num_engines: int = 1
     """number of vLLM Engines, set to 0 to disable vLLM"""
+    inference_batch_size: Optional[int] = None
+    """inference batch size per vLLM engine. If None, calculated as ceil(num_unique_prompts_rollout / vllm_num_engines) * num_samples_per_prompt_rollout"""
```

Comment:
could we also add an arg thats like `enable_queue_dashboard`? In case end user doesnt want the dashboard.